server/work-queue/queue.js

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var nr = require('node-resque');
+var Q = require('q');
+
+var config = require('../config');
+var log = require('../logger')('server.work-queue');
+var jobs = require('./jobs');
+
+var queueReady = Q.defer();
+var workersReady = Q.defer();
+
+var connectionDetails = {
+  host: config.get('db.redis.host'),
+  password: config.get('db.redis.password'),
+  port: config.get('db.redis.port'),
+  database: config.get('db.redis.database')
+};
+
+var queueOpts = { connection: connectionDetails };
+var queue = new nr.queue(queueOpts, jobs, function onQueueReady() {
+  log.info('queue started');
+  // TODO listen for queue events?
+  queueReady.resolve();
+});
+
+var multiWorkerOpts = {
+  connection: connectionDetails,
+  queues: ['chronicle'],
+  minTaskProcessors: 1,
+  maxTaskProcessors: 10,
```

Comment:
Trailing comma here.
Plus, not sure if `10` counts as a magic number and should maybe be moved into a Convict config for easy, environment-based tweakage.